be/src/common/config.h

```diff
@@ -462,7 +457,6 @@ CONF_Int64(load_process_max_memory_limit_bytes, "107374182400"); // 100GB
 CONF_Int32(load_process_max_memory_limit_percent, "30");         // 30%
 CONF_Bool(enable_new_load_on_memory_limit_exceeded, "false");
 CONF_Int64(compaction_max_memory_limit, "-1");
-CONF_Int32(compaction_max_memory_limit_percent, "100");
```

Comment:
The compaction_max_memory_limit_percent is used to limit the percent used.

Reply:
Don't remove it, we will change it to one small value if the compaction task support better scheduler

Reply:
okay, i will reserve it.